frontend/src/components/UserForms/TextInput.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import styles from '../UserTables/table.module.css';
+
+type InputProp = {
+  labelName: string,
+  labelText: string,
+  feedback: string,
+  showFormFeedback: string,
+  handleInput: ((e: any) => void)
+}
+
+const TextInput = (
+  { labelName, labelText, feedback, showFormFeedback, handleInput }: InputProp
+) => {
+  return (
+    <div className={styles.formDiv}>
+      <label htmlFor={labelName} className={styles.formLabel}>
+        {labelText}
+      </label >
+      <input type="text"
+        name={labelName}
+        onChange={(e) => handleInput(e)} />
+      <p className={showFormFeedback}>
+        {feedback}
+      </p>
```

Comment:
Might be better to have this as a conditional render `{showFormFeedback && <p>{feedback}</p>}` or something like that. Removes the need to specify a class as that attribute, making the code easier to understand.

Reply:
That's a good point! I was just copying the original code and didn't realize it can be better!